pkg/compare/compare.go

```diff
@@ -555,18 +555,18 @@ func newSummary(reference *Reference, c *MetricsCorelatorDecorator, numDiffCRs i
 func (s Summary) String() string {
 	t := `
 Summary
-CRs with diffs: {{ .NumDiffCRs }}
+Diff count: {{ .NumDiffCRs }}
 {{- if ne (len  .RequiredCRS) 0 }}
 CRs in reference missing from the cluster: {{.NumMissing}} 
 {{ toYaml .RequiredCRS}}
 {{- else}}
-No CRs are missing from the cluster
+Successfully found all required reference resources from the cluster
 {{- end }}
 {{- if ne (len  .UnmatchedCRS) 0 }}
 Cluster CRs unmatched to reference CRs: {{len  .UnmatchedCRS}}
 {{ toYaml .UnmatchedCRS}}
 {{- else}}
-No CRs are unmatched to reference CRs
+Successfully compared all reference resources
```

Comment:
The message can be misleading, If there are entries in this section it means that there were no CRs in the cluster that don't have a match in the reference CRs
i think  if we want to phrase it in a positive way it should be : "Successfully found a match for all cluster CRs to reference CRs" 
(something like this , the phrasing\grammar can be changed but that is the idea) 

Anyway i think we should remove the section and include it only when using -A flag. when not using this flag this section will not be empty only if there is a CR that had multiple matches to refernce CRs, most of the time it will be empty and not clear enough. So its confusing, i will open some issues to address this behavior .